app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,44 @@
+@import url('https://fonts.googleapis.com/css?family=Open+Sans');
+
+.body {
+  font-family: 'Open Sans';
```

Comment:
you should add a default font like sans-serif in case this font doesn't load `font-family: 'Open Sans', sans-serif;`